flintrock.py

```diff
@@ -573,7 +635,7 @@ def launch_ec2(
             tags={
                 'flintrock-role': 'slave',
                 'Name': '{c}-slave'.format(c=cluster_name)})
-
+        
```

Comment:
Style: Blank lines should not have any whitespace.

Travis (which currently just runs `lint.sh`) will flag problems like these for you, so they should be easy to find and fix. You can see this in the PR build status.